jdaviz/configs/specviz/plugins/unit_conversion/unit_conversion.py

```diff
@@ -42,16 +43,21 @@ class UnitConversion(PluginTemplateMixin):
     * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.close_in_tray`
     * ``spectral_unit`` (:class:`~jdaviz.core.template_mixin.SelectPluginComponent`):
       Global unit to use for all spectral axes.
-    * ``flux_unit`` (:class:`~jdaviz.core.template_mixin.SelectPluginComponent`):
+    * ``flux_or_sb_unit`` (:class:`~jdaviz.core.template_mixin.SelectPluginComponent`):
```

Comment:
Can you please add an entry for `flux_or_sb` and mention that it is only relevant/exposed for cubeviz?

Reply:
I have added this and will be included in the next commit